app/seeders/demo_data/projects_seeder.rb

```diff
@@ -74,5 +69,11 @@ def seedable_welcome_settings
         welcome_on_homescreen: 1
       }
     end
+
+    private
+
+    def project_seeders
+      @project_seeders ||= seed_data.each_data("projects").map { |project_data| ProjectSeeder.new(project_data) }
```

Comment:
minor but I like shorter lines
```suggestion
      @project_seeders ||= seed_data.each_data("projects")
                                    .map { |project_data| ProjectSeeder.new(project_data) }
```